.github/workflows/codecov_ats.yml

```diff
@@ -13,12 +13,6 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
-defaults:
-  run:
-    # the default default is:
-    #      bash --noprofile --norc -eo pipefail {0}
-    shell: bash --noprofile --norc -eo pipefail -ux {0}
-
```

Comment:
Please restore this?
It makes things much simpler to debug when things fail.
You mentioned running into an output-size limit: The below suggestion about `echo $(cmd)` may fix that.